### PR TITLE
portfolio website

### PR DESCRIPTION
**Pull request by khorh**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+ul,
+ol {
+  padding-left: 32px;
+  margin: 0;
+}
+
+/* navigation */
+nav {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (charliecog) [inline styles.css:106]: Not sure this is doing anything, potential for removal

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+}
+
+nav div a:nth-child(2) {
+  margin-left: 32px;
+}
+
+/* home */
+
+.hero {
+  display: flex;
```
Comment (charliecog) [inline styles.css:126]: Not sure this is doing anything, potential for removal

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+  flex-wrap: wrap;
```
Comment (charliecog) [inline styles.css:127]: Not sure this is doing anything, potential for removal

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+  justify-content: space-between;
```
Comment (charliecog) [inline styles.css:128]: Not sure this is doing anything, potential for removal

File: index.html [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    <nav>
+      <a href="index.html"><img src="images/logo.svg" alt="logo" /></a>
+      <div>
+        <a href="index.html">Projects</a>
+        <a href="about.html">About</a>
+      </div>
+    </nav>
+    <header>
+      <div class="hero">
+        <div>
```
Comment (charliecog) [inline index.html:28]: extra layer of div not required to produce the layout you have chosen

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+  font-weight: 500;
+  line-height: 1.5rem;
+}
+
+h3 {
+  font-size: 0.625rem;
+  font-weight: 500;
+  line-height: 1.25rem;
+}
+h4 {
```
Comment (charliecog) [inline styles.css:45]: missing line break above this line

**Review by charliecog — COMMENTED** [review]
A really brilliant build - not many bits to address - comments are detailed inline. 

A piece of general feedback:
on the project astra page there are some text characters that dont appear to be correct eg in the project brief the word 'clients'

File: styles.css (hunk 6 above)
Comment (khorh) [reply styles.css:45]: Updated!

File: styles.css (hunk 1 above)
Comment (khorh) [reply styles.css:106]: I have delete the flex-wrap property.  I was using it when my logo was longer.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <title>Helen's Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="normalize.css" type="text/css" />
+    <link rel="stylesheet" href="styles.css" type="text/css" />
```
[the remark this answers is not in the document]
Comment (khorh) [reply index.html:7]: Thanks!

File: about.html [hunk 8]
```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <title>Helen's Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="normalize.css" type="text/css" />
+    <link rel="stylesheet" href="styles.css" type="text/css" />
```
[the remark this answers is not in the document]
Comment (khorh) [reply about.html:7]: Done!

File: styles.css (hunk 2 above)
Comment (khorh) [reply styles.css:126]: No change.  It is more for responsive design.

File: styles.css (hunk 3 above)
Comment (khorh) [reply styles.css:127]: No change.  It is more for responsive design.

File: styles.css (hunk 4 above)
Comment (khorh) [reply styles.css:128]: No change.  It is more for responsive design.

File: index.html (hunk 5 above)
Comment (khorh) [reply index.html:28]: No change.  It is more for responsive design.

**Comment by khorh** [comment]
Completed changes as per code review.

**Comment by khorh** [comment]
Not sure what you mean by the text but then again I am very short sighted.

**Pull request merged**